Dfe.Academies.External.Web/Pages/School/FinancesReview.cshtml.cs

```diff
@@ -256,7 +252,7 @@ private FinancesReviewHeadingViewModel PopulateLoansHeading(SchoolApplyingToConv
 
 			var financesReviewHeading = new FinancesReviewSectionViewModel(
 					FinancesReviewSectionViewModel.ExistingLoans,
-					selectedSchool.Loans.Any() ? "Yes" : "No");
```

Comment:
why did you flip this logic back? as I flipped it as it wasn't working for me

Reply:
The HasLoans is a flag that Paul created for the loans summary page to distinguish between selecting no loans and not starting the page